pkg/node/local.go

```diff
@@ -758,7 +758,7 @@ func listLocalClusters(app *application.Avalanche, clusterNamesToInclude []strin
 	return localClusters, nil
 }
 
-func DestroyCurrentIfLocalNetwork(app *application.Avalanche) error {
+func CurrentIsLocalNetwork(app *application.Avalanche) (bool, string, error) {
```

Comment:
nit: add comment what `(bool, string, error)`  means in func return as it's not clear from the name

Reply:
maybe func name is not that great, there should be a better name like `GetLocalNetworkState` which returns clusterName and state running or not

Reply:
renamed to ConnectedToLocalNetwork.